### ISSUE TITLE
Make File.get_url_path() return a relative path

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#11594]
In #11032 `get_absolute_url()` implementation on our models was made to always return an absolute URL, and `get_url_path()` always return a relative URL...

... except in `File.get_url_path()`. Let's fix that, making things consistent. It will unblock several issues around links in the admin.

**Comment by diox** [comment]
QA:
- Once this lands, make sure add-on files can still be downloaded from reviewer tools, devhub etc, and installed from the frontend

**Comment by ioanarusiczki** [comment]
@diox I tested with a couple of add-ons, static themes, dictionaries , lang packs on AMO dev - FF67(Win10). 
- installation from the frontend works as expected. 
- downloads from rev tools and devhub work.
- while testing if file downloads  work from [Contents ](https://reviewers.addons-dev.allizom.org/en-US/firefox/files/browse/408934/) I've noticed that a simple click on .css, .txt, .js, .json files have a redirect such as https://reviewers.addons-dev.allizom.org/en-US/firefox/files/browse/408934/file-serve/README.txt?token=1481b920-51bd-49b7-bc6b-d309f4951ee8 or https://reviewers.addons-dev.allizom.org/en-US/firefox/files/browse/408934/file-serve/manifest.json?token=9a594976-c8ca-47ec-9e64-6e8de1ee2721. They can be downloaded right-clicking the mouse and then choosing "Save link as". Most of the file-types are automatically downloaded by just clicking the Download blue link which is displayed at the bottom of the page. However, what I described is the same way reproducible on AMO stage.
Let me know if that's an intended old behavior for the files types mentioned above.

An example - downloading an .aff and then the .json

![json2](https://user-images.githubusercontent.com/33448286/59200051-0c5d9a80-8ba0-11e9-9698-978dc348608e.gif)
